### ISSUE TITLE
Project rename?

### ISSUE THREAD

**Issue by LeeFrancis** [EBSCOIS/gulp-sassmaster#1]
Perhaps we could rename this to a project for all extensions..
something like electrode-extensions??

**Comment by pivnickm** [comment]
We certainly can. I was thinking that maybe we'd keep like things together, though. So CSS/SASS stuff would be separate from other Gulp task modules we create.

**Comment by LeeFrancis** [comment]
Yes.... but....  electrode is going to release SASS support soon as part of the webpack build. When that happens this project becomes unnecessary, unless we repurpose it.

**Comment by wylie** [comment]
So doesn't that make the name moot?

**Comment by wylie** [comment]
Speaking of naming things, the [stylelint](https://github.com/EBSCOIS/stylelint) repo is only temporary
